pipelines/internal/commands/run/run.go

```diff
@@ -207,7 +207,7 @@ func runPipeline(ctx context.Context, service *genomics.Service, req *genomics.R
 
 		lro, err := service.Pipelines.Run(req).Context(ctx).Do()
 		if err != nil {
-			if err, ok := err.(*googleapi.Error); ok {
+			if err, ok := err.(*googleapi.Error); ok && err.Message != "" {
```

Comment:
They will probably both be nill of non nill but even is the body is nill and the message is not it would be worth printing out the message instead of the plain error.